### DISCUSSION TITLE
function resolveDirectories takes too long

### DISCUSSION THREAD

**Discussion by 0-Captain** [sapegin/mrm#329] [Bugs]
https://github.com/sapegin/mrm/blob/master/packages/mrm/bin/mrm.js#L70
```
  console.time("resolveDir");
  const directories = await resolveDirectories(defaultDirectories);
  console.timeEnd("resolveDir");
```

print:
resolveDir: 1:00.790 (m:ss.mmm)

**Comment by ext** [comment]
Try installing `mrm-preset-default` globally, if it isn't present it will install it with `npx` (for the current process, so a new install is required the next time as well).

**Comment by zardoy** [comment]
> Try installing mrm-preset-default globally

Yes, it solves the problem.

> if it isn't present it will install it with npx

I would prefer `mrm` failing to long silent start (yo behavior). Would be better to be explicit at docs and point that the preset should be also installed with mrm.

**Comment by BurtHarris** [comment]
I just ran into this problem too, I had to fall back to a slow network connection.  

I did a little debugging, and noticed that **mrm-preset-default** is DEPRECATED!   Seem like there could be some optimization for this case.

**Comment by zardoy** [comment]
@sapegin  Did you just want the issue counter to be 0? Interesting move.
